lib/index.js

```diff
@@ -156,16 +156,16 @@ const nameSpaceIds = {
     Help: 12,
     HelpTalk: 13,
     Category: 14,
-    CategoryTalk: 15,
+    CategoryTalk: 15
 };
 nameSpaceIds.Image = nameSpaceIds.File;
 nameSpaceIds.ImageTalk = nameSpaceIds.FileTalk;
 
 /* Define a boolean is<ns> function for every namespace */
 for (const ns in nameSpaceIds) {
     if (Object.prototype.hasOwnProperty.call(nameSpaceIds, ns)) {
-        (function(id) {
-            Namespace.prototype[`is${ns}`] = function() {
+        (function (id) {
+            Namespace.prototype[`is${ns}`] = function () {
```

Comment:
since we are touching these can as well convert to arrow syntax

Reply:
Arrow syntax is discouraged with prototype